src/breadcrumbs/breadcrumbs.scss

```diff
@@ -18,47 +18,67 @@
   user-select: none;
 }
 
-@mixin iui-breadcrumbs-item {
+@mixin iui-breadcrumbs-item($breadcrumbs-height: $iui-component-height) {
```

Comment:
Plan to have different height?

Reply:
This was in prep to be used within header, but I'm going to remove it from this PR and add it in if I need it for the header because I'm uncertain atm.